sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/CallbackRegistration.java

```diff
@@ -28,18 +27,20 @@ public final class CallbackRegistration {
   private final ThrottlingLogger throttlingLogger = new ThrottlingLogger(logger);
   private final List<SdkObservableMeasurement> observableMeasurements;
   private final Runnable callback;
-  private final String callbackDescription;
```

Comment:
This `callbackDescription` field was really a `toString()` implementation in disguise.